core/src/com/unciv/ui/worldscreen/optionstable/WorldScreenOptionsTable.kt

```diff
@@ -77,6 +77,12 @@ class WorldScreenOptionsTable(val worldScreen:WorldScreen) : PopupTable(worldScr
             update()
         }
 
+        innerTable.add("Autocomplete turn".toLabel())
+        innerTable.addButton(if (settings.autoCompleteTurn) "Yes".tr() else "No".tr()) {
+            settings.autoCompleteTurn = !settings.autoCompleteTurn
+            update()
+        }
```

Comment:
I think this option should be in the _Gameplay options_ section.

Reply:
Oh yeah
Good point, missed that =)